Formula/open-image-denoise.rb

```diff
@@ -19,6 +19,9 @@ class OpenImageDenoise < Formula
   depends_on "tbb"
 
   def install
+    # Fix arm64 build targetting iOS
+    inreplace 'set(ISPC_TARGET_OS "--target-os=ios")', ""
```

Comment:
Think you need a filename here.

Reply:
Oops

Reply:
I look forward to the day `brew` is smart enough to know which file I want to `inreplace` without my specifying

Reply:
AI-powered formulae